v2/internal/controllers/generic_controller.go

```diff
@@ -142,8 +142,8 @@ func register(
 	clientFactory arm.ARMClientFactory,
 	info *registration.StorageType,
 	extensions map[schema.GroupVersionKind]genruntime.ResourceExtension,
-	options Options) error {
-
+	options Options,
```

Comment:
Is this the normal style we use? For some reason I thought we left `error` on the same line as the last line. Was this gofmt "fixing" things or you?

Reply:
yes, this was "fixed" 🙂